.github/workflows/windows_comp.yml

```diff
@@ -7,13 +7,13 @@
 
 name: Windows Compression GitHub CI
 
-on:
-  pull_request:
-    paths:
-      - 'crypto/comp/*.c'
-  push:
-    paths:
-      - '**.c'
+on: [pull_request]
+#  pull_request:
+#    paths:
+#      - 'crypto/comp/*.c'
```

Comment:
nit: Should we add .github/workflows/windows_comp.yml to the paths here so that we rebuild on pull request if the ci file changes in the future, to prevent issues like this?

Reply:
Done